scripts/common.py

```diff
@@ -625,6 +633,30 @@ def wait_for_port_to_be_open(port_number):
         sys.exit(1)
 
 
+def start_elasticsearch_server():
+    """Start the ElasticSearch server for running tests in development mode."""
+
+    # ElasticSearch server is only required in a development environment.
```

Comment:
Move this to the docstring above.

Also, the docstring above mentions "running tests". It's not just tests, right? Like, it's needed for regular operation of the local server too?

Reply:
Done